### ISSUE TITLE
grunt packaged not installed for wordpress-develop on first vagrant up

### ISSUE THREAD

**Issue by ericandrewlewis** [Varying-Vagrant-Vagrants/VVV#825]
The `grunt` package should be installed, as it's specified in [wordpress-develop's package.json](https://github.com/aaronjorbin/develop.wordpress/blob/master/package.json).

However, it is not installed on first `vagrant up`, although other packages are installed. 

To investigate, I made logs of `npm install` appear rather than be [swallowed to dev/null](https://github.com/Varying-Vagrant-Vagrants/VVV/blob/develop/provision/provision.sh#L698), and see [this output](https://gist.github.com/ericandrewlewis/9154837f5ba638d7ec76).

If I login to the machine after provisioning and run `npm install`, it installs `grunt` as well as `grunt-jsvalidate`.

```
vagrant@vvv:/srv/www/wordpress-develop$ npm install
WordPress@4.5.0 /srv/www/wordpress-develop
├─┬ grunt@0.4.5 
│ └── js-yaml@2.0.5 
└─┬ grunt-jsvalidate@0.2.2 
  └── esprima@1.0.4 
```

**Comment by jeremyfelt** [comment]
Good catch, I haven't tried using `grunt` inside the VM in a while.

It seems like the related issue could be this line: `UNMET PEER DEPENDENCY grunt@~0.4.0`.

I wonder if this is somehow related to #788 and #779.

**Comment by jeremyfelt** [comment]
I think we need to test this a bit more. I ran through `vagrant up` from scratch last week and `grunt` installed fine as part of provisioning.

**Comment by jeremyfelt** [comment]
I did another `vagrant up` with a blank box today as part of 1.3.0 testing and things seemed ok. I'm wondering if this was a one off issue.

**Comment by ntwb** [comment]
GruntJS 1.0.0 was released today http://gruntjs.com/blog/2016-04-04-grunt-1.0.0-released

Grunt-cli is also now 1.x, currently `1.2.0` https://github.com/gruntjs/grunt-cli

On a fresh VVV setup and an initial `vagrant up` and subsequent `vagrant up --provision` I've currently got:

```sh
vagrant@vvv:/srv/www/wordpress-develop$ grunt --version
grunt-cli v1.2.0
grunt v0.4.5
```

This all appears correct as Grunt `0.4.5` is what we currently install for WordPress ([src](https://core.trac.wordpress.org/browser/trunk/package.json#L14))

**Comment by GaryJones** [comment]
I'm not sure if this is related, but I recently *uninstalled global Grunt on my host machine* (preferring all projects to include their own dependencies), and I'm now getting the following:
![screenshot 2016-05-09 10 28 59](https://cloud.githubusercontent.com/assets/88371/15108979/e7c3480c-15d0-11e6-97d8-3d72b4b4b6a0.png)
After updating everything (VVV develop branch, latest box version, halted then `vagrant up --provision`), I then still got the same warning, but provisioning seemed to carry on:
![screenshot 2016-05-09 10 32 43](https://cloud.githubusercontent.com/assets/88371/15109070/6f77ccd2-15d1-11e6-80ed-1f7c0090a0cb.png)

There was an entry to say that Grunt was being installed, but even at the end, the error persisted:

![screenshot 2016-05-09 10 37 40](https://cloud.githubusercontent.com/assets/88371/15109247/6a1ee4ae-15d2-11e6-8bea-3973c2e63308.png)

**Comment by jeremyfelt** [comment]
I was able to reproduce this today on a fresh `vagrant up` with the develop branch. The [resulting npm-debug.log](https://gist.github.com/jeremyfelt/f1a48aaab92f734288f62ab0f7f15bdc) has a couple errors at the end that may be relevant.

It then took too many destroys and re-provisions, but I *think* I found how to reproduce.

In `provision.sh`, we have `noroot npm install &>/dev/null` for `wordpress-develop` on the first provision.

If I allow npm output to be shown by changing that to `noroot npm install` for a fresh provision, it works fine. If I change it back to `noroot npm install &>/dev/null` for a fresh position, it has the same error.

Now that I'm there, I haven't dug into possible reasons yet, but a short term fix may just be to allow for the output.
